lib/JSON/Schema/Fit.pm

```diff
@@ -134,7 +160,7 @@ sub _get_adjusted_boolean {
     my ($self, $struc, $schema, $jpath) = @_;
 
     return $struc  if !$self->booleans();
-    return JSON::true  if $struc;
+    return JSON::true  if $struc && $struc ne "false";
```

Comment:
This is slightly weird behavior, i don't think it should be default.
Better implement constructor option like this
`
my $jsfit = JSON::Schema::Fit(bool_resolve => sub {my $v = shift; !!$v && $v ne "false"});
`

Reply:
Indeed. That's why I pointed out it was a delicate situation. Reverted the change.
Will not implement resolvers for now. Thanks.